heat-stack/app/components/ui/heat/CaseSummaryComponents/AnalysisHeader.tsx

```diff
@@ -48,25 +48,35 @@ export function AnalysisHeader({
 	const summaryOutputs = usageData?.heat_load_output;
 
 	const totalRecords = usageData?.processed_energy_bills?.length || "-"
+	// Each record has three options for Analysis Types:
```

Comment:
@dwindleduck We should probably eventually consolidate these docs with https://github.com/codeforboston/home-energy-analysis-tool/blob/main/heat-stack/types/index.ts#L149-L153